src/views/pages/authentication/auth-forms/AuthRegister.js

```diff
@@ -1,55 +1,34 @@
 import { useState, useEffect } from "react";
-import { Link, useNavigate } from "react-router-dom";
-import { useSelector } from "react-redux";
+import { useNavigate } from "react-router-dom";
+// import { useSelector } from "react-redux";
 
```

Comment:
Remove changes for file from this PR, since they are not related to individual group page